bundles/io/org.eclipse.smarthome.io.console.rfc147/src/main/java/org/eclipse/smarthome/io/console/rfc147/internal/Rfc147Activator.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.io.console.rfc147.internal;
+
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+/**
+ *
+ * @author Markus Rathgeb - Initial contribution and API
+ *
+ */
+public class Rfc147Activator implements BundleActivator {
+
+    private static volatile Rfc147Manager manager;
```

Comment:
Why do you need this manager besides ConsoleSupportRfc147? Wouldn't the one service be enough? Is it because you need to get hold of the bundle context? If so, please note that you can also get hold of this through the ComponentContext: https://osgi.org/javadoc/r4v42/org/osgi/service/component/ComponentContext.html#getBundleContext()